modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/apiclient-header.mustache

```diff
@@ -13,7 +13,7 @@
 #include "{{packageName}}/ApiException.h"
 #include "{{packageName}}/IHttpBody.h"
 #include "{{packageName}}/HttpContent.h"
-
+{{^hasModelImport}}#include "{{packageName}}/ModelBase.h"{{/hasModelImport}}
```

Comment:
please use the following instead to avoid blank lines
```
{{^hasModelImport}}
#include "{{packageName}}/ModelBase.h"
{{/hasModelImport}}
```

Reply:
Fixed in https://github.com/OpenAPITools/openapi-generator/pull/21235/commits/3c051e8c2a531c2a39c546a8ca91fb13d561f723